### PR TITLE
Implement the transaction and command for revoke and self nomination

### PR DESCRIPTION
**Pull request by foriequal0**

### CODE REVIEW

**Comment by sgkim126** [comment]
@foriequal0 How about merging https://github.com/CodeChain-io/codechain-stakeholder-sdk-js/pull/8 first and add optional fee argument to theses commits too?

**Review by sgkim126 — COMMENTED** [review]
LGTM. @majecty How about you?

File: bin/cmds/selfNominate.ts [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    })
+};
+
+async function printSummary(
+    sdk: SDK,
+    blockNumber: number,
+    account: PlatformAddress,
+    cccChanges?: U64
+) {
+    console.group("Account", account.value);
```
Comment (majecty) [inline bin/cmds/selfNominate.ts:110]: Wow. I saw it for the first time.

**Review by majecty — APPROVED** [review]
LGTM

**Review by sgkim126 — APPROVED** [review]

**Pull request merged**